### ISSUE TITLE
Module (AssetManager) could not be initialized

### ISSUE THREAD

**Issue by xFran** [zfcampus/zf-apigility-skeleton#129]
Hi, 

After running $ php composer.phar update, now I get this error:

```
Fatal error: Uncaught Zend\ModuleManager\Exception\RuntimeException: Module (AssetManager) could not be initialized. in /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-modulemanager/src/ModuleManager.php:203 Stack trace: #0 /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-modulemanager/src/ModuleManager.php(175): Zend\ModuleManager\ModuleManager->loadModuleByName(Object(Zend\ModuleManager\ModuleEvent)) #1 /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-modulemanager/src/ModuleManager.php(97): Zend\ModuleManager\ModuleManager->loadModule('AssetManager') #2 [internal function]: Zend\ModuleManager\ModuleManager->onLoadModules(Object(Zend\ModuleManager\ModuleEvent)) #3 /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-eventmanager/src/EventManager.php(490): call_user_func(Array, Object(Zend\ModuleManager\ModuleEvent)) #4 /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-eventmanager/src/EventManager.php(251): Zend\EventManager\EventManager->triggerListeners('loadModules', Object(Zend\Mo in /var/www/zoo/apigility/zoo-api/vendor/zendframework/zend-modulemanager/src/ModuleManager.php on line 203
```

Any idea why and how can I fix it?

Thank you.

**Comment by xFran** [comment]
Somehow the AssetManager gets uninstalled. Why?
Anyways, run **$ php composer.phar require rwoverdijk/assetmanager** to fix it.

**Comment by webimpress** [comment]
@xFran because of [zf-apigility 1.3.0 release](https://github.com/zfcampus/zf-apigility/releases/tag/1.3.0)

**Comment by delboy1978uk** [comment]
I just got that too. Isn't this a BC break?

**Comment by weierophinney** [comment]
It's a change in dependencies, in order to achieve the same functionality. This was a necessary break, due to the fact that, at the time, rwoverdijk/assetmanager was going to take some months to be updated, and we didn't want to block the Apigility 1.3.0 release due to a 3rd party dependency.

We _did_ document the change, and the application changes necessary to accommodate the change; additionally, we provided a [comprehensive migration document](https://apigility.org/documentation/intro/updating-to-1-4) for users who wanted to upgrade to that release in order to take advantage of the ZF3 features.

**Comment by delboy1978uk** [comment]
Hi Matthew

Thanks for the detailed explanation, and the migration document! 
However I guess I should also apologise, as I now realise what I had done!

I use Git Bash and Cygwin on my work laptop, and both are hooked up to PHP, one to the latest, and one to v5.3.29 which my work is running on. But I probably installed it in the old terminal! When I ran update from the new one the other day, it realised it was a later PHP and updated everything, breaking my code. 

I used the temporary fix of requiring it back in, but now I can follow the document and migrate it properly! Thanks!

PHP versions don't usually suddenly change, so it's probably a very edge case that I ran into and not many people would experience that!

Thanks once again!
Derek
